src/sonic-config-engine/minigraph.py

```diff
@@ -493,7 +503,11 @@ def parse_xml(filename, platform=None, port_config_file=None):
             name = 'eth' + str(mgmt_intf_count)
             mgmt_intf_count += 1
             mgmt_alias_reverse_mapping[alias] = name
-        results['MGMT_PORT'][name] = {'alias': alias, 'admin_status': 'up'}
+        if port_speeds_default.has_key(alias):
+            port_speed = port_speeds_default[alias]
+            results['MGMT_PORT'][name] = {'alias': alias, 'admin_status': 'up', 'speed': port_speed}
```

Comment:
>'speed': port_speed} [](start = 80, length = 20)

suggest append it conditionally. if we have more optional field, the if-else doesn't work well.